_pages/about-us/offices/distributed.md

```diff
@@ -36,49 +36,49 @@ title: Distributed
 
 _Before you do anything, read our [best practices for making distributed teams work](https://18f.gsa.gov/2015/10/15/best-practices-for-distributed-teams/)._
 
-## <a id="how-do-i">How do I&hellip;</a>
+## How do I&hellip;
```

Comment:
This will may also break without manual control, but I'm not sure. We'd need to test it, I think.